packages/calendar/src/index.ts

```diff
@@ -7,7 +7,12 @@ import {
   CalendarConfigExternal as CalendarConfig,
   CustomComponentFn,
 } from '@schedule-x/shared/src/interfaces/calendar/calendar-config'
-import { CalendarEvent } from '@schedule-x/shared/src'
+import {
+  CalendarEvent,
```

Comment:
I think you will probably need `toDateTimeString` too :)

Reply:
The one converts into full-day events with the format YYYY-MM-DD, and the other into timed events with the format YYYY-MM-DD HH:mm

Reply:
Oh it was hidden haha, ill add that real quick.